packages/cx/src/ui/adapter/ArrayAdapter.js

```diff
@@ -90,7 +90,7 @@ export class ArrayAdapter extends DataAdapter {
          store: recordStore,
          index: index,
          data: data,
-         type: "data",
+         type: DataAdapterRecordType.Data,
```

Comment:
I don't see DataAdatperRecordType defined in the `DataAdapter.js` file. It's not enough to add it to the `.d.ts` file only.